src/runtime_src/core/common/CMakeLists.txt

```diff
@@ -94,8 +94,14 @@ install(TARGETS xrt_coreutil
   RUNTIME DESTINATION ${XRT_INSTALL_BIN_DIR}
 )
 
-install(TARGETS xrt_coreutil xrt_coreutil_static
+install(TARGETS xrt_coreutil
   EXPORT xrt-dev-targets
   ARCHIVE DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT}
   LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT} ${XRT_NAMELINK_ONLY}
 )
+
+install(TARGETS xrt_coreutil_static
+  EXPORT xrt-static-targets
+  ARCHIVE DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_STATIC_COMPONENT}
+  LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_STATIC_COMPONENT} ${XRT_NAMELINK_ONLY}
+)
```

Comment:
I don't understand why we would need a static package / target?

Reply:
I think I answered this in response to your first comment at https://github.com/Xilinx/XRT/pull/6830#discussion_r912500947